data/ar/4.0.dict

```diff
@@ -2,9 +2,8 @@
 %
 % Copyright (c) 2006 Warren Casbeer and Jon Dehdari
 %
-% Dictionary version number is 5.0.1 (formatted as V5v0v1+)
-<dictionary-version-number>: V5v0v1+;
-<dictionary-locale>: C+; % Only the English alphabet is in use here
+#define dictionary-version-number 5.0.1;
```

Comment:
Should be 5.9.0 -- all dicts with this change will be 5.9.0, since, obviously, these dicts are no longer compatible with older versions of LG ... older versions of the binary will error out, if used with new dicts.

Reply:
I added a commit for that.